packages/replayio/src/commands/record.ts

```diff
@@ -144,10 +140,8 @@ async function record(url: string = "about:blank") {
 
         console.log(""); // Spacing for readability
       } else {
-        const defaultRecording = findMostRecentPrimaryRecording(nextRecordings);
-
         selectedRecordings = await selectRecordings(nextRecordings, {
-          defaultSelected: recording => recording === defaultRecording,
+          defaultSelected: recording => recording.metadata.processType === "root",
```

Comment:
Default select all "root" recordings seems like the right thing to do anyway

Reply:
That's fine.